meson.build

```diff
@@ -83,6 +83,10 @@ foreach header : ['pwd.h', 'strings.h', 'netinet/in.h', 'sys/socket.h']
   endif
 endforeach
 
+if cxx.get_id() == 'msvc'
+  conf_data.set('NOMINMAX', true)
```

Comment:
Suggest adding a comment in the code that this is an MSVC-recognized flag that prevents it from shdowing the C++ std::min/max commands.  Just for maintainers who haven't dealved into these painful annoyances w/ MSVC. Otherwise, they might think its something our own code recognizes.